src/git/from_vcs.py

```diff
@@ -136,7 +136,8 @@ def git_pieces_from_vcs(tag_prefix, root, verbose, runner=run_command):
     else:
         # HEX: no tags
         pieces["closest-tag"] = None
-        count_out, rc = runner(GITS, ["rev-list", "HEAD", "--count"], cwd=root)
+        count_out, rc = runner(GITS, ["rev-list", "HEAD", "--left-right"], cwd=root)
+        count_out = len(count_out.split())
         pieces["distance"] = int(count_out)  # total number of commits
```

Comment:
`len()` is already an `int`, and `count_out` seems like a misapplied variable name, now.

```suggestion
        out, rc = runner(GITS, ["rev-list", "HEAD", "--left-right"], cwd=root)
        pieces["distance"] = len(out.split())  # total number of commits
```